commit 8a3ed1df9f75a591d6b216c680fedcdd643b490c
Author: Michael Cyr <mikecyr@us.ibm.com>
Date:   Fri Mar 17 00:48:20 2017 +0000

    scsi: ibmvscsis: Issues from Dan Carpenter/Smatch
    
    [ Upstream commit 11950d70b52d2bc5e3580da8cd63909ef38d67db ]
    
    Signed-off-by: Michael Cyr <mikecyr@us.ibm.com>
    Signed-off-by: Bryant G. Ly <bryantly@linux.vnet.ibm.com>
    Tested-by: Steven Royer <seroyer@linux.vnet.ibm.com>
    Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1416_8a3ed1df9f75a591d6b216c680fedcdd643b490c.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1416_8a3ed1df9f75a591d6b216c680fedcdd643b490c.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
